app/templates/400.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
```

Comment:
These error pages are largely the same. I suggest we put the common parts in a template and reuse them. That way these pages are tiny and a change to the design doesn't need to be replicated in all pages.

Reply:
Okay, I'll use template inheritance here